dll/win32/winmm/playsound.c

```diff
@@ -92,7 +92,21 @@ static HMMIO	get_mmioFromProfile(UINT uFlags, LPCWSTR lpszName)
      *      HKCU\AppEvents\Schemes\Apps\.Default
      *      HKCU\AppEvents\Schemes\Apps\<AppName>
      */
-    if (RegOpenKeyW(HKEY_CURRENT_USER, wszKey, &hRegSnd) != 0) goto none;
+    err = RegOpenCurrentUser(KEY_READ, &hUserKey);
+    if (err != ERROR_SUCCESS)
+    {
+        goto none;
+    }
+
+    err = RegOpenKeyW(hUserKey, wszKey, &hRegSnd);
+
+    RegCloseKey(hUserKey);
+
+    if (err != ERROR_SUCCESS)
+    {
+        goto none;
+    }
```

Comment:
Could be simplified (as you did in the other function) by:
```suggestion
    if (err == ERROR_SUCCESS)
    {
        err = RegOpenKeyW(hUserKey, wszKey, &hRegSnd);
        RegCloseKey(hUserKey);
    }

    if (err != ERROR_SUCCESS)
    {
        goto none;
    }
```

Reply:
OK.